src/pages/Article.js

```diff
@@ -92,17 +93,26 @@ const SeriesLabel = styled(SectionLabel)({
 export default connect('content')(({ content }) => (
     <article>
         <Header />
-        <Labels>
-            <SectionLabel>The NSA files</SectionLabel>
-            <SeriesLabel>Glenn Greenwald on security and liberty</SeriesLabel>
-        </Labels>
-        <Headline>{content.headline}</Headline>
-        <Standfirst
-            dangerouslySetInnerHTML={{
-                __html: content.standfirst,
-            }}
-        />
-
+        <Grid>
+            <Row>
+                <Col wide={4} leftCol={4}>
```

Comment:
how do you feel about `Cols`, rather than `Col`? i feel like the component is really the unit of the value in the prop in this one? no biggie though

Reply:
I see your point and I feel rather good about it